content/src/ports/activeEntities.ts

```diff
@@ -94,6 +101,47 @@ export function createActiveEntitiesComponent(
 
   const normalizePointerCacheKey = (pointer: string) => pointer.toLowerCase()
 
+  const cache = new LRU<string, Entity | NotActiveEntity>({
+    max: components.env.getConfig(EnvironmentConfig.ENTITIES_CACHE_SIZE)
+  })
+  const fixedCache = new Map<string, Entity | NotActiveEntity>()
+
+  const createLRUandFixedCache = (maxLRU: number, fixedTypes: EntityType[]) => {
+    return {
+      get(entityId: string) {
+        return cache.get(entityId) || fixedCache.get(entityId)
+      },
+      set(entityId: string, entity: Entity | NotActiveEntity) {
+        const isFixed = fixedCache.has(entityId) || (typeof entity !== 'string' && fixedTypes.includes(entity.type))
```

Comment:
Ain't the second part of the condition enough to use the fixedCache? (I mean if entity type is SCENE we should already store it regardless if it is already stored there, right?)